sites/docs/pages/getting-started.md

```diff
@@ -0,0 +1,163 @@
+---
+sidebar_position: 3
+hide_table_of_contents: false
+title: Build your first dashboard
+description: A 10 minute guide to building your first dashboard in Evidence.
+---
+<div id="objectives" class="alert svelte-17118v7">
+<strong>By the end of this 10 minute guide, you will be able to:</strong>
+<ul>
+   <li>Create and edit a page in Evidence</li>
+   <li>Write a query on demo data</li>
+   <li>Create a table</li>
+   <li>Create a chart</li>
+   <li>Connect a new CSV data source</li>
+</ul>
+</div>
+
+
+<Alert status="info">
+Casey note: I like how the Rails tutorial calls out what you'll achieve, and adds these satisfying green checkmarks. I think it makes the reader feel good and excited. Baked my own CSS for this, happy to change to your suggestions
+</Alert>
+
+## Prerequisites
+
+Please ensure that you have already installed Evidence: [Install Evidence](/install-evidence).
+
+New to web development? Start with the [Basics](#basics).
+
+Familiar with running a server at `localhost`, and writing pages in Markdown? Skip to [Working with data](#working-with-data).
+
```

Comment:
Should we explain about there being two ways to use Evidence?

> You can control Evidence via VSCode, or directly from the command line. Throughout this tutorial we'll include both options. If you're unsure, start with VSCode

Reply:
Oh interesting that you phrase it like this. In my mind the "Start Evidence" button is a one-click shortcut for typing `npm exec evidence dev -- --open /` in the command line. it's just a different way of starting the server at localhost:3000.

The intention was that anyone who is already familiar with the idea of running a local web app doesn't need to be babysat thru how to get all this started, and probably wants to skip straight to the data stuff. But maybe I'm not seeing all this with "non-dev" eyes so to speak

If the vast majority of your users think of these as two different "ways" to use Evidence, I can rewrite the intro with that perspective in mind.

<img width="1211" alt="image" src="https://github.com/evidence-dev/evidence/assets/16248601/b2d9d23b-5578-4f75-82d4-78c47dbfe134">

Reply:
it's correct that it's a shortcut, nothing special about it. Maybe "different ways" is the wrong terminology.

i'm easy to leave this out, was a relatively impulsive suggestion

Reply:
If that's how users mentally square it, happy to oblige. I changed the basics intro to this, but still tried to make it clear that what they are doing in either case is starting a dev server. Lemme know what u think!

<img width="785" alt="image" src="https://github.com/evidence-dev/evidence/assets/16248601/0ced46a5-724a-4f9e-8473-218d079adddf">

<img width="777" alt="image" src="https://github.com/evidence-dev/evidence/assets/16248601/937c59ba-af37-471b-940c-94d335af7fab">